po/fr.po

```diff
@@ -0,0 +1,130 @@
+# Turkish translation of io.github.bytezz.IPLookup.
+# Copyright (C) 2023 io.github.bytezz.IPLookup COPYRIGHT HOLDER
+# This file is distributed under the same license as the io.github.bytezz.IPLookup package.
+#
+# Sabri Ünal <libreajans@gmail.com>, 2023.
```

Comment:
The comments in the header should be changed ("Turkish translation" and "Sabri Ünal")